allennlp/training/trainer.py

```diff
@@ -166,6 +167,16 @@ def train(self) -> None:
                 metrics["loss"] = float(train_loss / batch_num)
                 description = self._description_from_metrics(metrics)
                 train_generator_tqdm.set_description(description)
+
+                batch_num_tot = num_training_batches * epoch + batch_num
+                if self._serialization_prefix and batch_num_tot % self._summary_interval == 0:
+                    for name, param in self._model.named_parameters():
+                        train_log.add_scalar("PARAMETER_MEAN/" + name, param.data.mean(), batch_num_tot)
+                        train_log.add_scalar("PARAMETER_STD/" + name, param.data.std(), batch_num_tot)
+                        if param.grad is not None:
+                            train_log.add_scalar("GRAD_MEAN/" + name, param.grad.data.mean(), batch_num_tot)
+                            train_log.add_scalar("GRAD_STD/" + name, param.grad.data.std(), batch_num_tot)
+                    train_log.add_scalar("LOSS/loss_train", metrics["loss"], batch_num_tot)
```

Comment:
This is already getting logged down below, so we probably don't need it here.  And on that note, can we put the code that does tensorboard logging together?

Reply:
The logging below is only per epoch, while this is per 100 batch so much more frequent. I agree we don't need both though.